councilmatic_core/management/commands/refresh_pic.py

```diff
@@ -59,6 +59,8 @@ def _get_urls(self):
                 INNER JOIN councilmatic_core_eventagendaitem as e_item
                 ON e_doc.event_id=e_item.event_id
                 WHERE e_item.updated_at >= (NOW() - INTERVAL '1 hour')
+                /* import_data runs up to four times per hour on the production site. */
+                /* Select items updated in the last hour, as a precaution for getting the most recently created ones. */
```

Comment:
more academic questions! is it possible for a document to be updated, without this timestamp being updated?

Reply:
(i'm trying to remember back to when this bug first occurred, and i can't remember whether we we'd registered the document had changed.)

Reply:
Good question! And, you know what, I think I missed an obvious way to determine if a document was updated – the Event model, which could have an updated date or name or whatever.